### PR TITLE
Add official docs

### PR DESCRIPTION
**Pull request by basepi**
Will need to have the docs add these to elastic/docs as well.

Ref #26

### CODE REVIEW

**Comment by basepi** [comment]
@bmorelli25 Can you help me get this included in elastic/docs?

**Comment by bmorelli25** [comment]
Hi @basepi. Sure! I opened https://github.com/elastic/ecs-logging-python/issues/27 to track all of the pieces.

**Comment by bmorelli25** [comment]
`elastic/docs` PR is up. I'm assuming you figured this out, as this PR builds without any errors, but others can test with this command:

```
$GIT_HOME/docs/build_docs --doc $GIT_HOME/ecs-logging-python/docs/index.asciidoc --resource=$GIT_HOME/ecs-logging/docs/ --chunk 1 --open
```

File: docs/setup.asciidoc [hunk 1]
```diff
@@ -0,0 +1,200 @@
+[float]
```
Comment (bmorelli25) [inline docs/setup.asciidoc:1]:
```suggestion
```

File: docs/setup.asciidoc [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+from the https://github.com/elastic/apm-agent-python[Elastic APM Python agent] in order to
+https://www.elastic.co/guide/en/apm/agent/python/current/log-correlation.html[correlate logs to spans, transactions and traces] in Elastic APM.
+
+[float]
+[[filebeat]]
+== Install Filebeat
+
+The best way to collect the logs once they are ECS-formatted is with https://www.elastic.co/beats/filebeat[Filebeat]:
+
+include::{ecs-repo-dir}/setup.asciidoc[tag=configure-filebeat]
```
Comment (bmorelli25) [inline docs/setup.asciidoc:200]: Thanks for using this!

File: .gitignore [hunk 3]
```diff
@@ -129,4 +129,7 @@ dmypy.json
 .pyre/
 
 # JUnit file
-junit-test.xml
\ No newline at end of file
+junit-test.xml
+
+# VSCode
+.vscode/
```
Comment (bmorelli25) [inline .gitignore:135]:
```suggestion
.vscode/

# Doc build
html_docs
```

**Comment by basepi** [comment]
Thanks for that build command. I mostly stole the template for this from Ruby so I can't take all the credit for it building successfully. :)

**Review by sethmlarson — APPROVED** [review]
Content looks good to me! :)

File: .gitignore [hunk 4]
```diff
@@ -129,4 +129,10 @@ dmypy.json
...
```
Comment (sethmlarson) [inline .gitignore:135]: non-review comment: Global gitignore files are where I store all my editor files https://docs.github.com/en/free-pro-team@latest/github/using-git/ignoring-files

**Review by bmorelli25 — APPROVED** [review]

File: .gitignore (hunk 4 above)
Comment (basepi) [reply .gitignore:135]: Not a bad idea. Thanks!

**Pull request merged**